### ISSUE TITLE
Unable to go get the sdk

### ISSUE THREAD

**Issue by shhsu** [Azure/azure-sdk-for-go#589]
I run into the same issue as
https://github.com/Azure/azure-sdk-for-go/issues/93

Same as @michaelsync, I tried
`go get github.com/Azure/azure-sdk-for-go`
`go get -u github.com/Azure/azure-sdk-for-go`
`go get -d github.com/Azure/azure-sdk-for-go`
`go get github.com/Azure/azure-sdk-for-go/...`

For the first 3 commands, I get the error message
`package github.com/Azure/azure-sdk-for-go: no buildable Go source files in c:\go\src\github.com\Azure\azure-sdk-for-go`

The last command yields
`# github.com/Azure/azure-sdk-for-go/arm/storageimportexport
arm\storageimportexport\client.go:62: undefined: acceptLanguage`

Note that I am using go 1.7.5.

**Comment by marstr** [comment]
Howdy @shhsu, I'm sorry you're running into trouble. 

The good news here is that despite the error message, `package github.com/Azure/azure-sdk-for-go: no buildable Go source files in c:\go\src\github.com\Azure\azure-sdk-for-go` you've acquired the SDK. The error message is stating that it is not able to continue on to pull down build dependencies or install the package into your $GOPATH/bin directory.

If you report the process targeting the exact package or set of packages that you need, the tool should exit quietly. For example, `go get -u github.com/Azure/azure-sdk-for-go/arm/compute` shouldn't produce any diagnostic messages.

However, you have pointed out a real problem, which is that apparently arm/storageimportexport is broken. Are you attempting to use `github.com/Azure/azure-sdk-for-go/storageimportexport`?

@jhendrixMSFT, @mcardosos, and @salameer and I will have to pow-wow to figure out what exactly happened with `storageimportexport`. I'll open a new issue to track that in particular.

**Comment by shhsu** [comment]
I tried
`go get -u  github.com/Azure/azure-sdk-for-go/arm/resources/subscriptions`
and the process exits without issue.

However, I am getting the same error when I `go build`
`can't load package: package github.com/Azure/azure-sdk-for-go: no buildable Go source files in c:\go\src\github.com\Azure\azure-sdk-for-go`

Also, I am not using `github.com/Azure/azure-sdk-for-go/storageimportexport`

**Comment by marstr** [comment]
I'm glad to hear you're not using `storageimportexport`, that eases this issue quite a bit.

The behavior that you describe above is the expected behavior of the `go get` tool and just telling you that `github.com/Azure/azure-sdk-for-go` is not an independent package. Rather, it is a repository that is host to 40+ packages. You have successfully acquired the SDK :) let us know if you have trouble using it.

**Comment by shhsu** [comment]
Getting independent packages indeed works. I am currently unblocked.
Would you like to close the issue or keep it open until this error is resolved when users tried to restore the whole bundle?
`can't load package: package github.com/Azure/azure-sdk-for-go: no buildable Go source files in c:\go\src\github.com\Azure\azure-sdk-for-go`

**Comment by marstr** [comment]
I'll close it out, thanks though @shhsu!
